molior/model/projectversion.py

```diff
@@ -98,7 +100,9 @@ def get_apt_repo(self, url_only=False, dist="stable"):
 
         if self.project.is_mirror:
             url = "{0}/{1}/mirrors/{2}/{3}".format(base_url, base_mirror, self.project.name, self.name)
-            full = "deb {0} {1} {2}".format(url, self.mirror_distribution, self.mirror_components.replace(",", " "))
+            # Workaround for aptly ('/' not supported as mirror dist)
+            full = "deb {0} {1} {2}".format(url, self.mirror_distribution.replace("/", "_-"),
```

Comment:
Why `_-` and not `-`?

Reply:
to not overlap with distributions which contain a real dash, and to make the translation backwards if needed...